### PR TITLE
Epic/widgets

### PR DESCRIPTION
**Pull request by digorath**
## Changes proposed in this PR:

This PR will accumulate all the changes for the widgets.
By itself it only provides the framework

---

SVA-80, SVA-82, SVA-83

### CODE REVIEW

File: src/components/widgets/Widgets.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  const filteredWidgets = getExistingWidgets(widgets);
+
+  if (!filteredWidgets?.length) return null;
+
+  const widgetComponents = filteredWidgets.map((Component, index) => {
+    return <Component key={index} navigation={navigation} />;
+  });
+
+  return <View style={styles.container}>{widgetComponents}</View>;
```
Comment (donni106) [inline src/components/widgets/Widgets.tsx:30]: we have several `Wrapper` styled components for container `View`s, which also makes sense here.
there is no existing wrapper for `row` and `space-between`, only a `WrapperRow`. i will create a `WrapperRowSpaceBetween` and use it here. so we do not need the `styles` and `StyleSheet` and `View` in this file.

Comment (donni106) [reply src/components/widgets/Widgets.tsx]: done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/154/commits/26661f8e177cf56258f9073ed6b8a217e7e19319

Comment (donni106) [reply src/components/widgets/Widgets.tsx]: even better is the ability to add a prop for the justify to not need to create multiple components for that.
like for the text component where several props can individualize the text component.
so we will have `<WrapperRow center>` and `<WrapperRow space-between>` and so on.

Comment (donni106) [reply src/components/widgets/Widgets.tsx]: done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/154/commits/7076d4c2a7eaf37592fe9ef50033565dff4c864b

Comment (digorath) [reply src/components/widgets/Widgets.tsx]: :+1:

**Review by donni106 — APPROVED** [review]

**Pull request merged**